vars/getChefServerCredentialsId.groovy

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/groovy
+
+def call(name) {
```

Comment:
Consider adding types, e.g.

```groovy
def call(String name) {
```

or even

```groovy
String call(String name) {
```

It's more code, but it makes it clear what kind of thing needs to be passed in, and what gets returned.